src/index.html

```diff
@@ -7,11 +7,158 @@
+        <input type="text"
+        name="usersurname"
+        autocomplete="off"
+        ><br>
+
+        <label for="name">Name:</label>
+        <input type="text"
+        name="username"
+        autocomplete="off"
+        ><br>
+
+        <label for="old"> How old are You?</label>
+        <input type="number"
+        name="userage"
+        min="1"
+        max="100"
+        value="12"
+        autocomplete="off"
+        ><br>
+
+        <label for="old"> Full date of birth:</label>
+        <input type="date"
+        name="userbirth"
+        ><br>
+
+        <label for="old"> I accept the term of the agreement</label>
+        <input type="checkbox"
+        name="iacc"
+        ><br>
+      </fieldset>
+    </form>
+
+
+    <form tabindex="2">
+      <fieldset>
+        <legend>Registration</legend>
+        <label for="surname">E-mail:</label>
+        <input type="email"
+         name="email"
+         autocomplete="off"
+         placeholder="email@example.com"
+        ><br>
+
+        <label for="password">Password:</label>
+        <input type="password"
+        name="password"
+        autocomplete="off"
+        >
+      </fieldset>
+    </form>
+
+
+    <form tabindex="3">
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+
+        <label>
+          <label for="cats">Do you love cats?</label>
+          <input type="radio"
+           name="cat"
+           value="yes"
+          > Yes
+        </label>
+
+        <label>
+          <input type="radio"
+          name="cat"
+          value="no"
+          > No <br>
+        </label>
+
+        <label for="color">What is your favorit color?</label>
+        <input type="color"
+        name="impcolor"
+        ><br>
+
+        <label for="old">What time do you go to bed?</label>
+        <input type="time"
+        name="userbedtime"
+        ><br>
+
+        <div>
+          <label for="car">
+            What are your favorite brand of cars?
+          </label>
+          <select
+           name="car"
+           required
+           multiple
+           >
+            <option value="BMV">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select><br>
+        </div>
+
+
+
+        <label for="rate">How do you rate our work?</label>
+        <input
+          type="range"
+          name="rateOurWork"
+          minlength="0"
+          maxlength="10"
+          value="0"
+        ><br>
+
+
+
+      </fieldset>
+    </form>
+
+    <form tabindex="3">
+      <fieldset>
+        <legend>Additional info:</legend>
+        <div>
+
+          <label for="feedback">
+              Comments:
+            <textarea name="feedback"
+              autocomplete="off"
+            >
+            </textarea><br>
+          </label>
+        </div>
+
+
+        <div>
+          <label for="recomendation">
+          Would you recomend us?
+          </label>
```

Comment:
children should be at a lower level than parents (code styling)
```suggestion
          <label for="recomendation">
            Would you recomend us?
          </label>
```